src/main/java/steps/CreateAccountPageSteps.java

```diff
@@ -3,24 +3,10 @@
 import models.AccountModel;
 import org.junit.Assert;
 import org.openqa.selenium.By;
-import pages.createAccountPage.CreateAccountInput;
 import pages.createAccountPage.CreateAccountPage;
 
 import static engine.drivers.WebDriverFactory.getWebDriverInstance;
-import static pages.createAccountPage.CreateAccountPage.address1;
-import static pages.createAccountPage.CreateAccountPage.address2;
-import static pages.createAccountPage.CreateAccountPage.city;
-import static pages.createAccountPage.CreateAccountPage.company;
-import static pages.createAccountPage.CreateAccountPage.confirmPassword;
-import static pages.createAccountPage.CreateAccountPage.email;
-import static pages.createAccountPage.CreateAccountPage.fax;
-import static pages.createAccountPage.CreateAccountPage.firstName;
-import static pages.createAccountPage.CreateAccountPage.lastName;
-import static pages.createAccountPage.CreateAccountPage.loginName;
-import static pages.createAccountPage.CreateAccountPage.password;
-import static pages.createAccountPage.CreateAccountPage.regionState;
-import static pages.createAccountPage.CreateAccountPage.telephone;
-import static pages.createAccountPage.CreateAccountPage.zipCode;
+import static pages.createAccountPage.CreateAccountPage.*;
```

Comment:
@Fishgame1 What is better in this case? Importing each item separately or the whole content?